src/Console/Command/CleanMySQLDatabaseCommand.php

```diff
@@ -60,92 +57,106 @@ protected function configure(): void
+        array $config
+    ): int {
         $io = new SymfonyStyle($input, $output);
 
-        $statuses = explode(',', $input->getOption('status'));
-        $statuses = array_map('trim', $statuses);
-        $statuses = array_map('strtolower', $statuses);
+        $statuses = $this->initStatuses($input, $io);
 
-        $invert = $input->getOption('invert');
-        $config = Yaml::parse(file_get_contents($input->getOption('config')));
-        $pattern = $input->getOption('pattern');
+        $invert           = $input->getOption('invert');
+        $pattern          = $input->getOption('pattern');
         $branchNameFilter = $input->getArgument('branchname-filter');
 
         if (!isset(
             $config['MySQL']['username'],
             $config['MySQL']['password'],
             $config['MySQL']['hostname']
         )) {
-            throw new \RuntimeException('MySQL config is incorrect! Username, password and hostname required!');
+            throw new RuntimeException('MySQL config is incorrect! Username, password and hostname required!');
         }
 
-        $dbUser = $config['MySQL']['username'];
+        $dbUser     = $config['MySQL']['username'];
         $dbPassword = $config['MySQL']['password'];
-        $dbHost = $config['MySQL']['hostname'];
+        $dbHost     = $config['MySQL']['hostname'];
 
         try {
             $issueService = new IssueService(new ArrayConfiguration([
                 'jiraHost'     => $config['Jira']['hostname'],
                 'jiraUser'     => $config['Jira']['username'],
                 'jiraPassword' => $config['Jira']['password']
             ]));
-        } catch (JiraException|\Exception $e) {
+        } catch (JiraException|Exception $e) {
             $io->error($e->getMessage());
+
             return 1;
         }
 
         $io->title('Scan databases');
 
         $connectionString = sprintf('mysql:host=%s;', $dbHost);
-        $pdo = new \PDO($connectionString, $dbUser, $dbPassword);
-        $pdo->setAttribute(\PDO::ATTR_ERRMODE, \PDO::ERRMODE_EXCEPTION);
+        $pdo              = new PDO($connectionString, $dbUser, $dbPassword);
+        $pdo->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
         $databases = array_column($pdo->query('SHOW DATABASES')->fetchAll(), 'Database');
         $databases = array_filter(
             $databases,
-            function ($dbName) use ($pattern) {
+            static function ($dbName) use ($pattern) {
                 return preg_match($pattern, $dbName);
             }
         );
 
         $remove = [];
-        $notfound = [];
 
         foreach ($databases as $database) {
 
             $branchName = !empty($branchNameFilter)
                 ? str_replace($branchNameFilter, '', $database)
                 : $database;
 
+            $issue = null;
             try {
                 $issue = $issueService->get($branchName, ['fields' => ['status']]);
+            } catch (Exception $e) {
```

Comment:
Propably catching `Throwable` here would be a good thing I think.

Reply:
Actually, no, because we rethrow anyways if it is not a 404

Reply:
Yes, but you don't catch `Errors`